superset/tasks/scheduler.py

```diff
@@ -60,9 +61,10 @@ def scheduler() -> None:
 
 
 @celery_app.task(name="reports.execute")
-def execute(report_schedule_id: int, scheduled_dttm: datetime) -> None:
+def execute(report_schedule_id: int, scheduled_dttm: str) -> None:
     try:
-        AsyncExecuteReportScheduleCommand(report_schedule_id, scheduled_dttm).run()
+        scheduled_dttm_ = parser.parse(scheduled_dttm)
+        AsyncExecuteReportScheduleCommand(report_schedule_id, scheduled_dttm_).run()
```

Comment:
Why the extra `_` in the second argument? Bug?

Reply:
Nevermind, I can't read.

Reply:
but that surfaces on SQLite only, I'm converting a `str` to `datetime`, I don't like to change types and use the same var